components/alert/alert.js

```diff
@@ -102,19 +108,24 @@ class Alert extends LocalizeCoreElement(RtlMixin(LitElement)) {
 			.d2l-alert-text-with-actions {
 				padding-right: 0.9rem;
 			}
+			.d2l-alert-text-with-icon {
+				padding-left: 0.5rem;
```

Comment:
Reduced padding since the icon has some padding between the highlight bar already.